tasks/main.yml

```diff
@@ -1,4 +1,17 @@
 ---
+
+# If we set these as facts, they are reachable via hostvars
+- set_fact:
+    openvpn_intra_if: "{{ ansible_default_ipv4.interface }}"
+  when: openvpn_intra_if is not defined
+- set_fact:
+    openvpn_intra_ipv4: "{{ hostvars[inventory_hostname]['ansible_' + openvpn_intra_if]['ipv4'] }}"
+  when: openvpn_intra_ipv4 is not defined
+
+- set_fact:
+    openvpn_intra_cidr: "{{ (openvpn_intra_ipv4.network + '/' + openvpn_intra_ipv4.netmask) | ipaddr('net') }}"
```

Comment:
… which would make the reference to `.network` fail here.

Reply:
Nevermind, seems to be macOS-specific.

Reply:
Thanks @priteau